xdocs/usermanual/component_reference.xml

```diff
@@ -6720,13 +6719,34 @@ Both Chrome and Internet Explorer use the same trust store for certificates.
         By default it also removes <code>If-Modified-Since</code> and <code>If-None-Match</code> headers.
         These are used to determine if the browser cache items are up to date;
         when recording one normally wants to download all the content.
-        To change which additional headers are removed, define the JMeter property <code>proxy.headers.remove</code>
+        To change which additional headers are removed, define the JMeter property <code>proxy.heade'rs.remove</code>
```

Comment:
Seems to be a strange name for a property. Is `proxy.heade'rs` really correct?

Reply:
I don't change this line, i get it from a previous version before the modiifcation, but you are rigth, the correct <code>proxy.headers.remove</code>